.gitlab/source_test/ebpf.yml

```diff
@@ -48,7 +48,7 @@ tests_ebpf_x64:
     # Compile runtime security stress tests to be executed in kitchen tests
     - inv -e security-agent.build-stress-tests --output=$DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files/stresssuite
     # Compile master version for comparison, uncomment following lines when merged
-    - git checkout -f master
+    - git checkout -f main || git checkout -f master
```

Comment:
This can still fail if a pipeline started running before the rename but got to this point after the rename. I think it's fine if this fails on the day of the rename.